### ISSUE TITLE
A post comparing agent frameworks

### ISSUE THREAD

**Issue by andyk** [agentos-project/agentos#118]
https://datarevenue.com/en-blog/ml-platforms-dataiku-vs-alteryx-vs-sagemaker is the sort of article that indicates a maturing market/trend or possibly a paradigm. I think If we do a post comparing the RL and cog sci / neuroscience frameworks we know about, that would be the sort of thing that could help (in a small way) our agenda to progress the agent paradigm

**Comment by nickjalbert** [comment]
Yeah, feels like we've already done a fair bit of the work here, and it'd be satisfying to finalize (a v1) and post it somewhere.
